### ISSUE TITLE
Cannot find module: 'react-konami-code' with 1.2.2

### ISSUE THREAD

**Issue by despatates** [vmarchesin/react-konami-code#1]
I upgraded to ^1.2.2, and this broke my production build.

The error is:
```
> my-project@0.14.4 build-js /data
> react-scripts build

Creating an optimized production build...
Failed to compile.

./src/components/CustomLayout/CustomLayout.js
Cannot find module: 'react-konami-code'. Make sure this package is installed.

You can install this package by running: npm install react-konami-code.
```

However `react-konami-code` is set in my dependencies in `package.json`, and is in `node_modules` folder.
```
node_modules/react-konami-code:
total 24
-rw-r--r-- 1 deploy-build deploy-build  350 Oct 26  1985 CHANGELOG.md
-rw-r--r-- 1 deploy-build deploy-build 1072 Oct 26  1985 LICENSE
-rw-r--r-- 1 deploy-build deploy-build 3740 Oct 26  1985 README.md
drwxr-xr-x 2 deploy-build deploy-build 4096 Nov 30 17:51 dist
-rw-r--r-- 1 deploy-build deploy-build 2219 Nov 30 17:51 package.json
drwxr-xr-x 3 deploy-build deploy-build 4096 Nov 30 17:51 test
```

Any idea about this error?

Your last commit is:
> Fix third party vulnerabilities

How risky is it to continue using v1.2.0? Cause it works fine.

**Comment by vmarchesin** [comment]
You are... absolutely correct. I accidentally published my package without the dist files, that's totally my bad. 

If you check your folder `node_modules/react-konami-code` you'll see that there is no `dist` folder. I fixed this by republishing the package under 1.2.3. Remove your `node_modules` folder and perform a fresh `npm install`, it should work now.

As for using 1.2.0 there shouldn't be any major issues unless your project has sensitive data. Either way I recommend using the latest version because of the vulnerability fixes.

Do not use version 1.2.2 since it is broken. The changelog correctly reflects it now.

Please close this issue if you confirm that it is working right now, I'll leave it open until then.

Thanks for the heads up by the way!

**Comment by despatates** [comment]
I confirm, problem solved with 1.2.3 :ok_hand: 

Thank you!
